core/templates/pages/exploration-editor-page/services/graph-data.service.ts

```diff
@@ -53,6 +43,16 @@ angular.module('oppia').factory('GraphDataService', [
       recompute: function() {
         _recomputeGraphData();
       },
+      /**
+       * @return graphData - Directed graph visualization input. This object
+       * includes the following keys:
+       * - nodes: Objects with keys of nodeids and values of node names.
+       * - links: list of objects. Each object represents a directed link
+       * between two nodes, and has keys 'source' and 'target', the values
+       * of which are the names of the corresponding nodes.
```

Comment:
nit: Please fix the alignment here so that it matches the way it was earlier. Thanks!

Reply:
Done